app/views/job_applications/new.html.erb

```diff
@@ -1 +1 @@
-<%= render 'form' %>
\ No newline at end of file
+<%= hello %>
```

Comment:
Development testing code?

Reply:
Correct - does this need to be removed to merge?